File: FirstProgram.c
#include<stdio.h>

int main(){

  const int T = 100;
  int sum;

  sum = 0;
  
  // Simple accumulation loop
  // (Note in Fortran we can use a built-in for this, but
  // ignore that for this demo)

  for(int i = 1; i <= T; i++){
    sum = sum + i;
  }

  printf("Loop gave         %i\n", sum);
  printf("Answer should be  %i\n", T*(T+1)/2);

}

File: FirstProgram.f90
PROGRAM MAIN

  USE omp_lib

  INTEGER, PARAMETER :: T=10000
  INTEGER :: sum
  INTEGER :: i

  sum = 0
  
  ! Simple accumulation loop
  ! (Note in Fortran we can use a built-in for this, but
  ! ignore that for this demo)

  DO i = 1, T
    sum = sum + i
  END DO

  PRINT*, "Loop gave        ", sum

  PRINT*, "Answer should be ", T*(T+1)/2

END PROGRAM
